api/v1/dois/PKPDoiHandler.php

```diff
@@ -33,12 +33,16 @@
 use PKP\security\authorization\RoleBasedHandlerOperationPolicy;
 use PKP\security\authorization\UserRolesRequiredPolicy;
 use PKP\security\Role;
+use PKP\services\PKPSchemaService;
 
 use Slim\Http\Request as SlimRequest;
 use Slim\Http\Response;
 
 class PKPDoiHandler extends APIHandler
 {
+    public const OBJECT_TYPE_PUBLICATION = 'publication';
+    public const OBJECT_TYPE_REPRESENTATION = 'representation';
```

Comment:
I think we need to consider chapter and submission file type in OMP

Reply:
Yes, I've opted to use the preexisting constants in the DOI repository, e.g. `Repo::doi()::TYPE_PUBLICATION`.